content/open-source/wolfi/building-a-wolfi-package.md

```diff
@@ -260,41 +260,73 @@ update:
+```
+
+This will create an ephemeral container based on the Wolfi SDK image, with a few predefined settings. We'll call this your **Wolfi development environment**.
+
+```
+❯ make dev-container
+docker run --privileged --rm -it \
+    -v "/home/erika/Projects/os:/home/erika/Projects/os" \
+    -w "/home/erika/Projects/os" \
+    -e SOURCE_DATE_EPOCH=0 \
+    ghcr.io/wolfi-dev/sdk:latest@sha256:99babbe4897d68ec1a342bd958fda7274a072bf112670fa691f64753b04774a9
+
+Welcome to the development environment!
+
+
+[sdk] ❯
 ```
 
-Once you have the key, you can run the `build` command. A few explanations before you hit the keyboard:
+You are now ready to build your Wolfi package.
 
-- **Building as root:** building packages require you to run melange as root, meaning you must include the `--privileged` parameter.
-- **Volumes:** here we set up **two** volumes: the current folder is shared with the /work location in the container, so that the packages generated will be available in your local system at the ./packages location. The second volume shares the `/tmp/melange_out` location in the container with your `/tmp` folder, so that you can have access to melange's working files even when the build fails. This helps a lot with debugging.
-- **Architecture:** limit the build to your own system's architecture while you are developing the package, this will make the build faster and avoid issues.
-- **Signing key:** remember this key will be required later as well, when you install the package in a container image using apko.
+### Building a Package
 
-Here is the full command:
+To build a package, run the following command from your Wolfi SDK environment:
 
 ```shell
-docker run --privileged --rm -v "${PWD}":/work -v /tmp/melange_out:/tmp \
-  cgr.dev/chainguard/melange build my-package.yaml \
-  --arch x86_64 \
-  --signing-key melange.rsa
+make package/<your-package-name>
 ```
 
+For instance, to build PHP 8.3, which is defined in a file named `php-8.3.yaml`, you would run `make package/php-8.3`:
+
+```
+[sdk] ❯ make package/php-8.3
+make yamlfile=php-8.3.yaml pkgname=php-8.3 packages/x86_64/php-8.3-8.3_rc3-r1.apk
+make[1]: Entering directory '/home/erika/Projects/os'
+##############################################################
+#          build output - removed for readability            #
+##############################################################
+ℹ️            | signing apk index at packages/x86_64/APKINDEX.tar.gz
+ℹ️            | signing index packages/x86_64/APKINDEX.tar.gz with key local-melange.rsa
+ℹ️            | appending signature to index packages/x86_64/APKINDEX.tar.gz
+ℹ️            | writing signed index to packages/x86_64/APKINDEX.tar.gz
+ℹ️            | signed index packages/x86_64/APKINDEX.tar.gz with key local-melange.rsa
+make[1]: Leaving directory '/home/erika/Projects/os'
+[sdk] ❯
+```
+
+When the build is finished, you can find the newly built apk(s) in a `./packages` directory, in the root of your cloned Wolfi repository.
+
```

Comment:
Again, can't put this below (when the build fails). But, I'd like there to be at least a little bit of description about these directories and what they mean:
```
2023/10/21 00:26:40 ERROR: failed to build package. the build environment has been preserved:
ℹ️  aarch64   |   workspace dir: /tmp/melange-workspace-2581732429
ℹ️  aarch64   |   guest dir: /tmp/melange-guest-2340770039
```

Meaning, where would you look for the log files that pertain to build vs. "output" and so forth.